ntc_templates/templates/alcatel_aos_show_port-security.textfsm

```diff
@@ -0,0 +1,23 @@
+Value PORT ((\d+\/?)+)
+Value OPERATION_MODE (\S+)
+Value ADMIN_STATE (\S+)
+Value MAX_MAC_BRIDGED (\d+)
+Value TRAP_THRESHOLD (\S+)
+Value MAX_MAC_FILTERED (\d+)
+Value VIOLATION_TYPE (\S+)
+Value VIOLATION_MAC (\S+)
+Value PKT_RELAY (\S+)
+
+
+Start
+  ^\s*Port:\s*${PORT}$$
+  ^\s*Admin-State\s*:\s*${ADMIN_STATE},$$
+  ^\s*Operation\s*Mode\s*:\s*${OPERATION_MODE},$$
+  ^\s*Max\s*MAC\s*bridged\s*:\s*${MAX_MAC_BRIDGED},$$
+  ^\s*Trap\s*Threshold\s*:\s*${TRAP_THRESHOLD},$$
+  ^\s*Max\s*MAC\s*filtered\s*:\s*${MAX_MAC_FILTERED},$$
+  ^\s*Violation\s*:\s*${VIOLATION_TYPE},$$
+  ^\s*Violating\s*MAC\s*:\s*${VIOLATION_MAC}\s*$$
+  ^\s*Pkt-Relay\s*:\s*${PKT_RELAY}$$ -> Record
+  ^\s*
```

Comment:
```suggestion
  ^\s*$$
```
I almost didn't catch this, and looking back we have some cleanup to do. This should look for lines that just contain space or are empty lines. This will require more updates to match at least the `----+` type lines